app/javascript/components/application_form/preferences/PreferencesSection.js

```diff
@@ -39,7 +45,10 @@ const PreferencesSection = ({ formApi, listingPreferences, editValues }) => {
         editPreference["naturalKey"] = naturalKey
         autofillPreferences.push(editPreference)
       }
+      console.log(preference)
```

Comment:
hi @fddayan let's get rid of all the `console.log` -- how were you able to commit this? lol. Did you install overcommit?